spec/lib/common/client/base_spec.rb

```diff
@@ -1,16 +1,15 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require './spec/support/default_configuration_helper'
```

Comment:
Thoughts on moving this into [`rails_helper.rb`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/rails_helper.rb#L9-L21)?